docs/source/release_notes.rst

```diff
@@ -17,6 +17,7 @@ Future Release
     * Changes
         * Remove framework for unused ``woodwork`` CLI (:pr:`1288`)
         * Add back support for Python 3.7 (:pr:`1292`)
+        * Nested statistical utility functions into directory (:pr:`1295`)
```

Comment:
Need to add your user name to the list of contributors at the bottom of this section.